### PR TITLE
Erik/feat/5 add condition3

### LINKED ISSUE #5: Condition 3 method

### PR DESCRIPTION
**Pull request by Storsorken**
This fixes issue 5 by implementing condition 3 and adds a static helper method to the Point class.

### CODE REVIEW

File: decide/src/main/java/org/example/InitialSettings.java [hunk 1]
```diff
@@ -72,5 +72,9 @@ public Point(int x, int y) {
         public double distance(Point other) {
             return Math.sqrt(Math.pow(x - other.x, 2) + Math.pow(y - other.y, 2));
         }
+
+        public static double triangleArea(Point p1, Point p2, Point p3) {
+            return Math.abs(p1.x * (p2.y - p3.y) + p2.x * (p3.y - p1.y) + p3.x * (p1.y - p2.y));
```
Comment (hampfh) [inline decide/src/main/java/org/example/InitialSettings.java]: From what I can tell you're using `Shoelace formula` here which seems to be a good fit for the problem but if I'm not mistaken the result should be divided by 2 at the end.


triangleArea((0,0),(1,0),(1,1)) => 1 (Correct 1/2)
triangleArea((0,0),(10,0),(10,10)) => 100 (Correct 50)

**Review by hampfh — CHANGES_REQUESTED** [review]
Nice work! Only have one question about the triangleArea implementation which I'd like to address

File: decide/src/main/java/org/example/InitialSettings.java (hunk 1 above)
Comment (Storsorken) [reply decide/src/main/java/org/example/InitialSettings.java]: Thanks for the catch!

**Review by hampfh — APPROVED** [review]
🌟

**Pull request merged**